API.md

```diff
@@ -0,0 +1,81 @@
+# Octoliner API
+
+## `class Octoliner`
+
+Create an object of type `Octoliner` to communicate with a particular Octoliner board.
+
+### `Octoliner(uint8_t i2caddress = 42)`
+
+Constructs a new object. The `i2caddress` specifies the board I²C address which is 42 factory-default but can be changed programmatically.
+
+### `void begin()`
+### `void begin(TwoWire* wire)`
+
+Initializes given interface, prepares the board for communication. If the `wire` argument is omitted, the connection is started on hardware I2C#0. Method is initialise selected or default TwoWire interface. If TwoWire interface was initialise before this method reinitialise it.
+
+Call this method once, in your `setup()`.
+
+### `void setSensitivity(uint8_t sense)`
+
+Change load resistance value for IR sensors. An increase in load resistance leads to decrease IR sensor sensitivity. 
+`0` corresponds to the minimal sensitivity, and `255` corresponds to the maximal sensitivity.
+
+For Amperka Dragster (6mm Octoliner to track distance) you can use values [230; 250]. If Octoliner closer to track you will be forced reduce value.
+
+### `int analogRead(uint8_t channel)`
```

Comment:
```suggestion
### `uint_16t analogRead(uint8_t channel)`
```

Otherwise, an inconsistency between 8-bit (int = 2 bytes) and 32-bit (int = 4 bytes) MCU arrises